docs/insights/complexity.md

```diff
@@ -22,7 +22,9 @@ This sniff
     <summary>Configuration</summary>
 
 ```php
-
+\NunoMaduro\PhpInsights\Domain\Insights\CyclomaticComplexityIsHigh::class => [
```

Comment:
:thinking: You added this in the comment template I added for add new Insight. This should be placed at L12